src/main/java/org/gridsuite/modification/server/impacts/AbstractBaseImpact.java

```diff
@@ -45,4 +46,10 @@ public enum ImpactType {
     private ImpactType type;
 
     private IdentifiableType elementType;
+
+    @JsonIgnore
+    public abstract boolean isSimple();
+
+    @JsonIgnore
+    public abstract boolean isCollection();
```

Comment:
Why?

Reply:
Asked by @SlimaneAmar for simplification